Source/Core/Core/PowerPC/Jit64/Jit.cpp

```diff
@@ -330,7 +330,7 @@ void Jit64::Init()
   EnableBlockLink();
 
   jo.fastmem_arena = m_fastmem_enabled && Memory::InitFastmemArena();
-  jo.optimizeGatherPipe = true;
+  jo.optimizeGatherPipe = false;
```

Comment:
I'm not entirely sure why this is needed, but assuming it is needed, you should probably only set this to false when your new setting is enabled.

Also, this needs to be changed in JitArm64 as well.

Reply:
It's not needed, I've fixed it now